### DISCUSSION TITLE
Chocolatey package

### DISCUSSION THREAD

**Discussion by teknowledgist** [standardnotes/forum#2693] [Suggestions]
Just FYI:  A [Standard Notes Chocolatey package](https://chocolatey.org/packages/standardnotes) has been created/submitted.  It should be approved in a few days.

**Comment by teknowledgist** [comment]
Scratch that.  This is installing only for the current user.  Is there any way to install for the system (i.e. all users)?  A lot of NSIS installers use [multiuser.nsh](https://nsis.sourceforge.io/Docs/MultiUser/Readme.html) which allows for the `/allusers` switch.

Any chance of including that?

**Comment by atmoio** [comment]
We use electron-builder, so we'd have to see if they support that.

**Comment by teknowledgist** [comment]
Being ignorant about Electron-Builder, I would guess [from this](https://www.electron.build/configuration/nsis) that it is possible.  

A system/AllUser install might have to be the default (which it should be anyway), so non-admin users would have to click in another dialog window.  Who cares?  System installs are most likely what users familiar with silent installs will want anyway.

Thanks.

**Comment by Yolakalemowa** [comment]
Hi are there plans to provide a chocolatey packages of SN? I couldn't find one listed.
